core/src/test/scala/org/apache/spark/deploy/security/KafkaTokenUtilSuite.scala

```diff
@@ -78,6 +80,21 @@ class KafkaTokenUtilSuite extends SparkFunSuite with BeforeAndAfterEach {
     Configuration.setConfiguration(null)
   }
 
+  test("checkProxyUser with proxy current user should throw exception") {
+    val realUser = UserGroupInformation.createUserForTesting("realUser", Array())
```

Comment:
not a big deal but `Array()` -> `Array.empty[String]`. Ignore this if everything in this PR is ready